datatypes/src/operations/image/to_png.rs

```diff
@@ -91,17 +102,14 @@ mod tests {
 
     #[test]
     fn linear_gradient() {
-        let mut raster = Raster2D::new(
-            [2, 2].into(),
-            vec![0; 4],
-            None,
-            Default::default(),
-            Default::default(),
-        )
-        .unwrap();
+        let mut raster = Raster2D::new([2, 2].into(), vec![0; 4], None).unwrap();
 
-        raster.set_pixel_value_at_grid_index(&(0, 0), 255).unwrap();
-        raster.set_pixel_value_at_grid_index(&(1, 0), 100).unwrap();
+        raster
+            .set_pixel_value_at_grid_index(&[0, 0].into(), 255)
```

Comment:
didnt you want to aliminate the `into`s?

Reply:
yes but it is hard enough to get this working as it is.